Oracle customization - non upstream patch
Add Solaris 12 to the pkg(5) pkg provider

--- puppet-3.8.6/lib/puppet/provider/package/pkg.rb.orig        2016-04-19 13:58:08.682627507 -0700
+++ puppet-3.8.6/lib/puppet/provider/package/pkg.rb     2016-04-19 14:01:27.178984250 -0700
@@ -1,3 +1,7 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 require 'puppet/provider/package'
 
 Puppet::Type.type(:package).provide :pkg, :parent => Puppet::Provider::Package do
@@ -23,7 +27,7 @@
 
   confine :osfamily => :solaris
 
-  defaultfor :osfamily => :solaris, :kernelrelease => '5.11'
+  defaultfor :osfamily => :solaris, :kernelrelease => ['5.11', '5.12']
 
   def self.instances
     pkg(:list, '-H').split("\n").map{|l| new(parse_line(l))}

